### DISCUSSION TITLE
利用 Vercel 搭建属于自己的 ChatGPT | Alan Wang

### DISCUSSION THREAD

**Discussion by giscus** [3Alan/site#49] [Announcements]
# 利用 Vercel 搭建属于自己的 ChatGPT | Alan Wang

整理了一些当前比较流行且 UI 还不错的开源 ChatGPT 项目，可以在 Vercel 上部署，方便大家快速搭建属于自己的 ChatGPT，同时提供部署方法及安全建议，防止被有心人利用。

https://www.alanwang.site/blog/personal-chatgpt

**Comment by isharmla233** [comment]
感谢收集整理~ 顺带一提：您的文章大抵是被盗搬了 http://element-ui.cn/chatgpt/10548.html
